cpp/include/raft/neighbors/detail/cagra/compute_distance_vpq.cuh

```diff
@@ -115,7 +116,8 @@ struct cagra_q_dataset_descriptor_t : public dataset_descriptor_base_t<half, DIS
   template <uint32_t DATASET_BLOCK_DIM, uint32_t TEAM_SIZE>
   __device__ DISTANCE_T compute_similarity(const QUERY_T* const query_ptr,
                                            const INDEX_T node_id,
-                                           const bool valid) const
+                                           const bool valid,
+                                           const raft::distance::DistanceType metric) const
```

Comment:
Metric is not used here? If VPQ doesn't support InnerProduct we should add a `RAFT_EXPECT` check probably?

Reply:
Yes. It does not support it yet. I can add a RAFT_EXPECT.